js/Models/data.js

```diff
@@ -0,0 +1,183 @@
+{
```

Comment:
pls put these files into src/data folder. models we will reserve for real model classes not for raw test data

Reply:
sure :)